bounties/level-2/sep10-javascript-server.md

```diff
@@ -0,0 +1,69 @@
+# JavaScript SEP-0010 Server Reference Implementation
+
+## Award
+| Hunter | Reviewer | Type
+| :- | :- | :-
+| 1000 XLM | 1000 XLM | Capped (3)
```

Comment:
I think it should be more for both parties. The result of this bounty could end up being used by many readers of the SEP so it should be solid and being put up with the right effort. There are many conditions in the SEP that need to be a) implemented and b) thoroughly reviewed.

Reply:
Agreed. I feel like I have very little frame of reference for what is(n't) a fair award amount (for both hunter and reviewer). 1000XLM would be (at the moment) just about 300USD. Should I double it?